### DISCUSSION TITLE
font size

### DISCUSSION THREAD

**Discussion by dlance87** [mmistakes/minimal-mistakes#2680] [Q&A] [answered]
## Environment

- Minimal Mistakes version:
- Ruby gem or remote theme version:
- Jekyll version:
- Git repository URL:
- GitHub Pages hosted (if yes provide URL to site):
- Operating system:

**Comment by mmistakes** [comment, marked as answer]
For questions related to changing the theme's font size consult this issue and the many duplicate ones linked to it.

https://github.com/mmistakes/minimal-mistakes/issues/1352
